aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -161,6 +124,24 @@ For more information on hosting, see [Host in ASP.NET Core](xref:fundamentals/ho
 
 ### IIS options
 
+**In-process hosting model**
```

Comment:
OH ... one more thing here. This new content should be versioned for 2.2 or later.